generated/nidcpower/nidcpower/lcr_measurement.py

```diff
@@ -0,0 +1,305 @@
+                +-----------------------+-----------------------------------------------------+
+                | InstrumentMode.LCR    | The channel(s) are operating as an LCR meter.       |
+                +-----------------------+-----------------------------------------------------+
+
+            in_compliances (InCompliances): Indicates whether the output was in DC compliance and/or
+                AC compliance at the time the measurement was taken.
+
+            unbalanced (bool): Indicates whether the output was unbalanced at the time the
+                measurement was taken.
+        """
+        if data is not None:
+            if isinstance(data, struct_NILCRMeasurement):
+                self.channel = ""
+                self.vdc = data.vdc
+                self.idc = data.idc
+                self.stimulus_frequency = data.stimulus_frequency
+                self.ac_voltage = complex(data.ac_voltage_real, data.ac_voltage_imaginary)
+                self.ac_current = complex(data.ac_current_real, data.ac_current_imaginary)
+                self.z = complex(data.z_real, data.z_imaginary)
+                self.series_lcr = LCRMeasurement.LCR(
+                    inductance=data.ls, capacitance=data.cs, resistance=data.rs
+                )
+                self.parallel_lcr = LCRMeasurement.LCR(
+                    inductance=data.lp, capacitance=data.cp, resistance=data.rp
+                )
+                self.d = data.d
+                self.measurement_mode = enums.InstrumentMode(data.measurement_mode)
+                self.in_compliances = LCRMeasurement.InCompliances(
+                    dc=bool(data.dc_in_compliance), ac=bool(data.ac_in_compliance)
+                )
+                self.unbalanced = bool(data.unbalanced)
+            else:
+                for field_name, _ in LCRMeasurement._lcr_measurement_field_metadata:
+                    setattr(
+                        self,
+                        field_name,
+                        enums.InstrumentMode(data.measurement_mode)
+                        if field_name == "measurement_mode"
+                        else getattr(data, field_name),
+                    )
+        else:
+            for field_name, _ in LCRMeasurement._lcr_measurement_field_metadata:
+                setattr(
+                    self,
+                    field_name,
+                    enums.InstrumentMode(measurement_mode)
+                    if field_name == "measurement_mode"
+                    else locals()[field_name],
+                )
+
+    def get_z_magnitude_and_phase(self):
+        """get_z_magnitude_and_phase
+
+        Returns a tuple of (z_magnitude, z_phase) of this LCRMeasurement object.
+
+        Returns:
+            z_magnitude_and_phase (tuple of (float, float)):
+
+                - **z_magnitude** (float): The magnitude of the complex impedance, in ohms.
+                - **z_phase** (float): The impedance phase angle, in degrees.
```

Comment:
Are you sure `polar` doesn't return radians?
If this is wrong, how did it pass tests? Might be missing coverage.